client/src/reference-provider/parse.ts

```diff
@@ -49,19 +55,39 @@ function insertRefs (templateName: string, path: string, lineNrs: any[], allCall
 }
 
 export function parseFile (file: string, allVariablePathMaps: VariablePathMap) {
-    const variablePattern: RegExp = /([\w\d._]+)[\w\d._, #]*/gim;
+    const variablePattern: RegExp = /([\w\d._]+)[\w\d._, #]*/gim; // This regex matches too many unnecesary stuff, it can be checked at https://regexr.com/

```

Comment:
Very true lol. 
A lot of this stuff I got form examples online. Gotta love it when it works out like that.
Have a better pattern suggestion?

Reply:
I can provide some that might be good yeah will dm you